cron_validator/regexes.py

```diff
@@ -21,10 +21,10 @@
                                     "(?P<step>(\\*|[1-2]?[1-9]|[1-3]0|31)/([1-2]?[1-9]|[1-3]0|31))"))
 
 month_re = re.compile("{0}|{1}|{2}|{3}|{4}".format("(?P<all>\\*)",
-                                                   "(?P<specific>[0-9]|1[0-1])",
-                                                   "(?P<range>([0-9]|1[0-1])-([0-9]|1[0-1]))",
-                                                   "(?P<list>([0-9]|1[0-1])(,([0-9]|1[0-1]))+)",
-                                                   "(?P<step>(\\*|[0-9]|1[0-1])/([1-9]|1[0-1]))"))
+                                                   "(?P<specific>[0-9]|1[0-2])",
```

Comment:
I think you should remove 0 on regex as well. At the beginning, I think I intent to use 0-11 for month. But it should be 1-12.

Reply:
@vcoder4c Good catch, I didn't think of that.. New commit is coming!